source/GStreamerMSEMediaPlayerClient.cpp

```diff
@@ -199,6 +199,7 @@ void GStreamerMSEMediaPlayerClient::notifySourceStartedSeeking(int32_t sourceId)
             auto sourceIt = m_attachedSources.find(sourceId);
             if (sourceIt == m_attachedSources.end())
             {
+                m_seekOngoing = true;
```

Comment:
Why is it added only when source couldn't be added? 
I would either set it every time or would change the name of this variable describe the situation more precisely.

Reply:
changed.